### DISCUSSION TITLE
Dropzone incorrect behavior

### DISCUSSION THREAD

**Discussion by dmitry-ff** [mantinedev/mantine#7136] [Q&A] [answered]
I have Dropzone compoenent which is accepts "application/zip" MIME type.
If i upload file by opening explorer > chosing files > press choose button everything is okey
but if i choose files in explorer and move them by mouse into dropzone area - onReject triggers

p.s. i am working only with zip archives

**Comment by rtivital** [comment, marked as answer]
It might be fixed in next patch 7.14.2, if not, you can report this issue in react-dropzone repository
